### ISSUE TITLE
Terraform 0.11.4 /Win64 don't see current state

### ISSUE THREAD

**Issue by yauhen-vastraknutau-epam** [hashicorp/terraform#17653]
### Terraform Version

```
Terraform v0.11.4
+ provider.aws v1.11.0
```

### Terraform Configuration Files

```hcl
provider "aws" {
  access_key = "bla-bla-bla"
  secret_key = "bla-bla-bla"
  region     = "us-east-1"
}

resource "aws_instance" "example" {
  ami           = "ami-b374d5a5"
  instance_type = "t2.micro"
}
```

### Expected Behavior

Expected that Terraform will refresh current infrastructure state and do nothing at the 3 step (repeated apply)

### Actual Behavior

Terraform don't see provisioned at the 2 step infrastructure and create a new one. State file after step 2 looks correct. After step 3 state file has been rewritten with a new infrastructure. 

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`
3. `terraform apply`
### Additional Context

Have tried to reproduce this under CentOS vm - non-reproducible, correct behavior there.

**Comment by jbardin** [comment]
Hi @yauhen-vastraknutau-epam 

Thanks filing the issue. Duplicate of #17620

**Comment by yauhen-vastraknutau-epam** [comment]
Thanks for a quick answer.
